sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRegistryAsyncClient.java

```diff
@@ -26,7 +26,7 @@
 import static com.azure.core.util.FluxUtil.withContext;
 
 /**
- * Initializes a new instance of the asynchronous ContainerRegistry type.
+ * Instantiates Container Registry async client.
```

Comment:
It's recommended to include codesnippets on how to create an instance of this client in the javadoc (use codesnippets). Generally, code snippets to create a client should be included in the builder, sync and async client classes' javadocs.

Reply:
I have not added it to this builder but the other one - just because there is conversation of moving away from 2 builders to 1 so want to add doc for this only if needed :)